tests/pytorch/sparse/test_elementwise_op.py

```diff
@@ -2,18 +2,22 @@
 import sys
 
 import backend as F
+
+import dgl.sparse
```

Comment:
`import dgl.sparse as sp`?

Reply:
`import dgl.sparse as dglsp`

Let's follow the conversion.

```
./tests/pytorch/sparse/test_reduction.py:import dgl.sparse as dglsp
./examples/sparse/sign.py:import dgl.sparse as dglsp
./examples/sparse/sgc.py:import dgl.sparse as dglsp
./examples/sparse/gcnii.py:import dgl.sparse as dglsp
./examples/sparse/c_and_s.py:import dgl.sparse as dglsp
./examples/sparse/hypergraphatt.py:import dgl.sparse as dglsp
./examples/sparse/appnp.py:import dgl.sparse as dglsp
./examples/sparse/gcn.py:import dgl.sparse as dglsp
./examples/sparse/graph_transformer.py:import dgl.sparse as dglsp
./examples/sparse/gat.py:import dgl.sparse as dglsp
./examples/sparse/hgnn.py:import dgl.sparse as dglsp
./examples/sparse/twirls.py:import dgl.sparse as dglsp
```